### ISSUE TITLE
Support Postman Collection Format v2 (and thus Insomnia)

### ISSUE THREAD

**Issue by pacud** [numberly/flask2postman#9]
Hey,

Could we have support for [insomnia](https://insomnia.rest/) please ?

**Comment by ramnes** [comment]
Yes! Insomnia already supports Postman collections, but we need to find out why it doesn't accept the collections exported by flask2postman.

**Comment by ramnes** [comment]
flask2postman exports Postman v1 collections, while Insomnia only support the Postman v2 schema. We should make the output configurable and support both.

https://schema.getpostman.com/json/collection/v2.0.0/docs/index.html

**Comment by ramnes** [comment]
Meanwhile, you can pipe the output of flask2postman to [postmanlabs/postman-collection-transformer](https://github.com/postmanlabs/postman-collection-transformer). :tada:
